.github/workflows/build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: 22
```

Comment:
What's the advantage of changing this version?

Reply:
It'll actually slow down builds because you'll install 17 only to then have to install 22.

Reply:
Yeah, totally correct. I changed it only to prove that the setup is now independent of what JDK is running the Gradle process.